kolibri/utils/version.py

```diff
@@ -228,19 +228,21 @@ def get_git_describe():
     :returns: None if no git tag available (no git, no tags, or not in a repo)
     """
     repo_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-    p = subprocess.Popen(
-        "git describe --tags --match 'v[0-9]*'",
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        shell=True,
-        cwd=repo_dir,
-        universal_newlines=True
-    )
-    # This does not fail if git is not available or current dir isn't a git
-    # repo - it's safe.
-    version_string = p.communicate()[0].rstrip()
-    return version_string
-
+    try:
+        p = subprocess.Popen(
+            "git describe --tags --match 'v[0-9]*'",
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            shell=True,
+            cwd=repo_dir,
+            universal_newlines=True
+        )
+        # This does not fail if git is not available or current dir isn't a git
+        # repo - it's safe.
+        version_string = p.communicate()[0].rstrip()
+        return version_string
+    except:
```

Comment:
Should avoid blanket `except` clauses. Catch the exception you are trying to catch.

If you are catching multiple exceptions `except (ValueError, IOError):`

Reply:
@rtibbles Updated!